unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreenV2/LoadingScreenV2Plugin.cs

```diff
@@ -1,14 +1,16 @@
 using DCL;
 using DCL.Controllers.LoadingScreenV2;
 using System.Collections.Generic;
+using UnityEngine;
 
 public class LoadingScreenV2Plugin : IPlugin
 {
-    private readonly HintRequestService controller;
+    private readonly LoadingScreenHintsController controller;
 
     public LoadingScreenV2Plugin()
     {
-        controller = new HintRequestService(new List<IHintRequestSource>(), new SceneController(), new HintTextureRequestHandler());
+        controller = new LoadingScreenHintsController((new GameObject().AddComponent<HintView>()),
```

Comment:
> new GameObject().AddComponent<HintView>()

This should be an addressable prefab.

Reply:
Another placeholder I left behind. I'll fix that too 😅